knative-operator/pkg/controller/knativeserving/servicemesh/servicemesh.go

```diff
@@ -161,7 +164,7 @@ func isServiceMeshControlPlaneReady(servingNamespace string, api client.Client)
 func installServiceMeshControlPlane(instance *servingv1alpha1.KnativeServing, api client.Client) error {
 	log.Info("Installing serviceMeshControlPlane")
 	const (
-		path = "deploy/resources/servicemesh.yaml"
+		path = "deploy/resources/serviceMesh/servicemesh.yaml"
```

Comment:
I think this is not "official" rule, but capital letter is not used in directory or file name in general. So, the directory should be `servicemesh` instead of `serviceMesh`?

Reply:
Yes, please :)